app/logic/programEligibility.py

```diff
@@ -9,14 +9,14 @@
 def isEligibleForProgram(program, user):
     """
     Verifies if a given user is eligible for a program by checking if they are
-    banned from a program and if they have attended all the required events for a program.
+    banned from a program.
 
     :param program: accepts a Program object or a valid programid
     :param user: accepts a User object or userid
     :return: True if the user is not banned and meets the requirements, and False otherwise
     """
-    program = Program.get_by_id(program)
-    if (ProgramBan.select().where(ProgramBan.user == user)) and (ProgramBan.select().where(ProgramBan.program == program)):
+
+    if (ProgramBan.select().where(ProgramBan.user == user, ProgramBan.program == program)):

```

Comment:
I'd add a .exists() to the end of this query to be explicit what we care about, since it is in an if statement.